microsoft-edge/webview2/concepts/working-with-local-content.md

```diff
@@ -476,6 +486,14 @@ When loading content via a file URL or a virtual host name mapping, the resoluti
 
 This can take some time. Make sure to limit calls to `AddWebResourceRequestedFilter` to only the web resources that must raise the `WebResourceRequested` event.
 
+<!-- ---------- -->
+###### Source maps with WebResourceRequested event
+
+Source maps are needed to debug the source code of compiled content like transpiled JavaScript (e.g. TypeScript, minified JavaScript) or CSS (e.g. SASS, SCSS). WebView2 does not load source maps referenced by content which was loaded using `WebResourceRequested` event. Consider the following example. You load JavaScript file `main.js` in your `WebResourceRequested` event handler by setting `WebResourceRequestedArgs.Response`. If `main.js` references `main.js.map` as its source map, then `main.js.map` will neither be loaded automatically nor your `WebResourceRequested` event handler will be called again to load it.
+
+To use source maps along with `WebResourceRequested`, choose one of the following approaches:
+- Generate inline source maps during compilation of your content. Inline source maps are embedded to the corresponding compiled file.
+- Inline separate source maps to the content at runtime in your `WebResourceRequested` event handler. Use this approach only if your content build system does not support inlining source maps.
```

Comment:
```suggestion
- Inline separate source maps to the content at runtime in your `WebResourceRequested` event handler. Use this approach only if your content build system does not support inline source maps.
```
@dhveerap - i think/assume you also want this instance changed from inlining to inline.  if i'm interp'g your other comment right.